### ISSUE TITLE
Intermediate pages

### ISSUE THREAD

**Issue by nwaxiomatic** [gadventures/django-fsm-admin#35]
Is it possible to add intermediate pages triggered by the state-changes?

Example case: changing state sends email, but allows you to add email content before sending in a new form.

**Comment by santiagomalter** [comment]
Good idea. It would also be useful if we want a confirmation for the action.
